Sources/Sentry/Public/SentryReplayApi.h

```diff
@@ -44,6 +44,42 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)stop;
 
+/**
+ * Shows an overlay on the app to debug session replay masking.
```

Comment:
`h`: If I'm not mistaken, this doesn't work for SwiftUI and only for UIKit? If yes, please mention this in the comment to clarify this.

Reply:
It does work for SwiftUI if used in runtime.